src/core/reactor.cc

```diff
@@ -4139,6 +4139,12 @@ void smp::configure(const smp_options& smp_opts, const reactor_options& reactor_
     }
 #endif
 
+#ifdef SEASTAR_DEFAULT_ALLOCATOR
+    if (!smp_opts.memory.defaulted()) {
+        seastar_logger.warn("Seastar compiled with default allocator, --memory limit is not supported");
+    }
+#endif
```

Comment:
I think this should be in the `if (smp_opts.memory) {` scope below.

Reply:
done